pod/perlclass.pod

```diff
@@ -0,0 +1,310 @@
+=head1 NAME
+
+perlclass - Perl class syntax reference
+
+=head1 SYNOPSIS
+
+    use v5.36;
+    use feature 'class';
+
+    class My::Example 1.234 {
+        field $x;
+
+        ADJUST {
+            $x = "Hello, world";
+        }
+
+        method print_message {
+            say $x;
+        }
+    }
+
+    My::Example->new->print_message;
+
+=head1 DESCRIPTION
+
+This document describes the syntax of the Perl's C<class> feature, which
+provider native keywords supporting object oriented programming paradigm.
```

Comment:
provides native keywords

Reply:
And better spelled as *object-oriented programming* (OOP) with a dash `-`

https://en.wiktionary.org/wiki/object-oriented_programming#English :-)